guides/creators.rst

```diff
@@ -0,0 +1,48 @@
+Using ESPHome for your Project
+==============================
+
+.. seo::
+    :description: Information for creators when using ESPHome firmware.
+
+There are a few configurtion options we have added to ESPHome to make it easier for creators
+to create, configure, install and distribute devices running ESPHome.
+
+Example configuration
+---------------------
+
+.. code-block:: yaml
+
+    esphome:
+      ...
+      name_add_mac_suffix: true
```

Comment:
Add inline comments next to each option describing what it does. Keep the links at the bottom. 
```suggestion
      # Automatically add the mac address to the name
      # so you can use a single firmware for all devices
      name_add_mac_suffix: true
```